definitions/variable/agriculture/agriculture.yaml

```diff
@@ -98,31 +153,46 @@
     unit: million t DM/yr
     agmip: Production (PROD) - ???
     tier: 3
-- Agricultural Production|Crops|Sugarcrops:
+- Agricultural Production|Crops|Oil crops:
+    definition: Production of crops
+    unit: million t DM/yr
+    agmip: Production (PROD) - ???
+    tier: 3
+- Agricultural Production|Crops|Sugar crops:
     definition: Production of crops
     unit: million t DM/yr
     agmip: Production (PROD) - Sugarcrops (SGC)
     tier: 3
-- Agricultural Production|Crops|Oilcrops:
+- Agricultural Production|Crops|Other crops:
     definition: Production of crops
     unit: million t DM/yr
     agmip: Production (PROD) - ???
     tier: 3
-- Agricultural Production|Crops|Energy Crops:
+- Agricultural Production|Crops|Energy crops:
     definition: Production of crops
     unit: million t DM/yr
     agmip: Production (PROD) - Energy crops (ECP)
     tier: 2
-- Agricultural Production|Crops|Other:
-    definition: Production of crops
-    unit: million t DM/yr
-    agmip: Production (PROD) - ???
-    tier: 2
 - Agricultural Production|Livestock:
-    definition: Production of crops
+    definition: Production of livestock products
     unit: million t DM/yr
     agmip: Production (PROD) - Livestock (LSP)
     tier: 2
+- Agricultural Production|Livestock|Dairy:
+    definition: Production of livestock products
+    unit: million t DM/yr
+    agmip: Production (PROD) - Livestock (LSP)
+    tier: 3
+- Agricultural Production|Livestock|Ruminant:
+    definition: Production of livestock products
+    unit: million t DM/yr
+    agmip: Production (PROD) - Livestock (LSP)
+    tier: 3
+- Agricultural Production|Livestock|Non-ruminant:
+    definition: Production of livestock products
```

Comment:
Please add an explanation what a non-ruminant is for an energy person like me...

Reply:
sure, I can copy-paste this from the demand definition